src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,13 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+function checkUser(user) {
+  if (user !== undefined) {
+    return `${user}`;
+  }
+
+  return '';
+}
```

Comment:
your should check if user exists in parent component. UserInfo should always accept user and render it. Use conditional rendering in parent component

Reply:
done